collectorsapp_2019-08-05.sql

```diff
@@ -0,0 +1,61 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 192.168.20.20 (MySQL 5.6.44)
+# Database: collectorsapp
+# Generation Time: 2019-08-05 13:22:34 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table grocery_item
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `grocery_item`;
+
+CREATE TABLE `grocery_item` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `item` varchar(255) DEFAULT NULL,
+  `category` enum('PANTRY','FRIDGE','FREEZER','PRODUCE','OTHERS') NOT NULL,
```

Comment:
The singular "other" might be better here

Will there be any further clarification as to what "other" means? If you have other as an option on a form it's usual to be able to add some free text in to describe what "other" means. <- Totally ok not to do this, just figured it was worth consideration

Reply:
ok